### DISCUSSION TITLE
AIP Browser DE -> Enroute

### DISCUSSION THREAD

**Discussion by Heinrich-Ffm** [Akaflieg-Freiburg/enroute#402] [Q&A]
Anflugkarten aus dem 'AIP Browser DE' nach Enroute kopieren.
Hier eine JSON, die bei mir funktioniert hat, wobei ich mich Frage ob ich die Pages in Enroute anzeigen kann, sonst könnte man den Bereich Pages wohl auch weglassen.

{
    "version": "1.0.2",
    "name":"AIP-VFR-Mitte",
    "description": "AIP-VFR Deutschland Mitte",
    "type": "NATIVE",
    "pages": {
        "coveredArea": [
            {
                "type": "GeoRectangle",
                "west": 6.0,
                "south": 49.0,
                "east": 12.0,
                "north": 51.0
            }
        ],
        "asSingleFile": false
    },
    "charts": {
        "coveredArea": [
            {
                "type": "GeoRectangle",
                "west": 6.0,
                "south": 49.0,
                "east": 12.0,
                "north": 51.0
            }
        ],
        "minInvScale":  0.0,
        "maxInvScale": 500000.0
    }
}

... und natürlich kann man die Koordinaten anpassen.

**Comment by kebekus** [comment]
Super, danke.

**Comment by DeltaTango69** [comment]
Pages habe ich weggelassen. So ist das ZIP auch einiges kleiner. So wie ich das aber verstehe werden die wohl in einer neuen Version auch angezeigt. Eventuell bei den Flugplätzen unter Bibliothek. Das Feld ist im Moment noch ausgegraut.
